android/app/src/main/java/com/bounswe2020group3/paperlayer/dagger/CollaborationRequestModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.bounswe2020group3.paperlayer.dagger
+
+import com.bounswe2020group3.paperlayer.collaborationRequests.CollabContract
+import com.bounswe2020group3.paperlayer.collaborationRequests.CollabModel
+import com.bounswe2020group3.paperlayer.collaborationRequests.CollabPresenter
+import dagger.Binds
+import dagger.Module
+
+@Module
+abstract class CollaborationRequestModule {
+
+    @Binds
+    abstract fun bindEventDetailPresenter(presenter: CollabPresenter): CollabContract.CollabPresenter
```

Comment:
Names of these functions doesn't imply the collaboration but it has no affect on the dependency injection. You can change them later.